jobs/integr8ly/ocp4/tests/ocp4-automated-namespace-recreate.yaml

```diff
@@ -0,0 +1,71 @@
+---
+
+- job:
+    name: ocp4-automated-namespace-recreate
+    description: 'Tests the integreatly operator to ensure any integreatly namespaces are recreated after deletion.'
+    project-type: pipeline
+    sandbox: true
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE repository containing the test source code.'
+      - string:
+          name: BRANCH
+          default: 'v2-master'
+          description: 'Branch of the repository'
+      - string:
+          name: CLUSTER_URL
+          description: 'URL of cluster on which the test will be executed, including port number.'
+      - string:
+          name: ADMIN_USERNAME
+          description: 'Username to login to Openshift 4 cluster. Provided by the feedback from installation.'
```

Comment:
the `Provided by the feedback from installation` is not necessarily true when using PDS clustersa

Reply:
Changes made to refer to PDS email